EmerisHQ/demeris
# fix: move locales to json

## Description
Moving all of our copy into a json file.

TLDR: This is quite urgent, because it's needed for the chrome extension to be audited:
https://allinbits.slack.com/archives/C02U9SVJT97/p1648917256436929

Long story:
> the problem with vue-i18n if anyone else wants to tackle it is that  the messages have to be precompiled because the runtime compiler uses eval which is blocked in extensions by CSP...
In order however to be able to use the precompiling vite plugin, the messages must be in json or yaml format....we currently have them as .ts files

Logic wise nothing should have changed

##How to test
Just go through the app.
Maybe especially check some copy that has variables inside of them
